samples/dynamically_add_module_sample/src/main.c

```diff
@@ -11,49 +11,102 @@ int main(int argc, char** argv)
     if (argc != 3)
     {
         printf("usage: dynamically_add_module_sample modulesConfigFile linksConfigFile\n");
-        printf("where configFile is the name of the file that contains the Gateway configuration\n");
+        printf("where modulesConfigFile is the name of the file that contains gateway modules and linksConfigFile is the name of the file that contains gateway links.\n");
+        printf("press ENTER to exit.\n");
```

Comment:
Ok, I will change. I don't agree but I will change. I think the experience on this sample is better than the others...

---
In reply to: [109019197](https://github.com/Azure/azure-iot-gateway-sdk/pull/189#discussion_r109019197) [](ancestors = 109019197)